### PR TITLE
adds UI for managing emails in the school configuration screen.

### PR DESCRIPTION
**Pull request by stopfstedt**
fixes https://github.com/ilios/ilios/issues/3357

### CODE REVIEW

File: packages/frontend/tests/integration/components/school/emails-editor-test.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    this.set('school', schoolModel);
+    this.set('cancel', () => {
+      assert.ok(true);
+    });
+    await render(
+      hbs`<School::EmailsEditor @school={{this.school}} @save={{(noop)}} @cancel={{this.cancel}} />`,
+    );
+    await component.cancel();
+  });
+});
```
Comment (jrjohnson) [inline packages/frontend/tests/integration/components/school/emails-editor-test.js:112]: optional - but maybe throw in a keybinding test for save and cancel?

**Review by jrjohnson — CHANGES_REQUESTED** [review]

File: packages/frontend/app/components/school/emails-editor.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      .trim()
+      .split(',')
+      .map((email) => email.trim())
+      .filter((email) => !isBlank(email));
+    return emails.reduce((valid, email) => EmailValidator(email) && valid, true);
+  }
+
+  @action
+  validateChangeAlertRecipientsMessageCallBack() {
+    return 'The change alerts recipients list contains invalid email addresses.';
```
[the remark this answers is not in the document]
Comment (stopfstedt) [reply packages/frontend/app/components/school/emails-editor.js:47]: doh! i did put the translations into the lang files.

*headdesk

fix incoming...

[the remark this answers is not in the document]
Comment (stopfstedt) [reply packages/frontend/app/components/school/emails-editor.js]: :heavy_check_mark:

File: packages/frontend/tests/integration/components/school/emails-editor-test.js (hunk 1 above)
Comment (stopfstedt) [reply packages/frontend/tests/integration/components/school/emails-editor-test.js:146]: :heavy_check_mark:

**Review by jrjohnson — APPROVED** [review]

**Comment by dartajax** [comment]
I can imagine a few things to make this even cooler but it totally does the job. Gonna merge momentarily,

**Pull request merged**